src/v0/staff.ts

```diff
@@ -147,11 +156,36 @@ export class Staff {
         response.status !== 200 &&
           reject(new errors.StaffDeleteFailed(undefined, { status: response.status }))
 
+        return resolve({ msg: response.data.msg } as StaffMemberResponse)
+      } catch (error) {
+        return reject(new errors.StaffDeleteFailed(undefined, { error }))
+      }
+    })
+  }
+
+  getPin(providedPin?: PinRequest): Promise<StaffMemberResponse> {
+    const queryString = qs.stringify(providedPin)
+    return new Promise(async (resolve, reject) => {
+      let uri = `${this.options.base}${this.endpoint}/${this.options.user}/pin`
+      if (queryString) {
+        uri = `${uri}?${queryString}`
```

Comment:
Just for the future. You can get rid adding `queryString` conditionally, by using the `addQueryPrefix` option for `stringify`.

from the `qs` docs:
> `assert.equal(qs.stringify({ a: 'b', c: 'd' }, { addQueryPrefix: true }), '?a=b&c=d');`

I've tested it, also works if the query object is empty or `undefined`. In such cases it stringifies to an empty string (i.e. no '`?`'), so it's safe to always be appended.

Reply:
Cool! Good to know.